i18n/tests/test_url_resolver.py

```diff
@@ -0,0 +1,36 @@
+from django.test import TestCase
+from django.contrib.auth.models import Permission
+from django.utils import translation
+
+from curricula.factories import UserFactory, CurriculumFactory, UnitFactory
+from lessons.factories import LessonFactory, ResourceFactory, ActivityFactory
+from django.conf import settings
+
+class I18nUrlResolverTestCase(TestCase):
+    def setUp(self):
+        self.csf_curriculum = CurriculumFactory(
+            slug="csf-curriculum",
+            unit_template_override='curricula/csf_unit.html')
+        self.pl_curriculum = CurriculumFactory(
```

Comment:
I'd love to see something explicitly explaining why we're testing with this curriculum in particular